course_catalog/serializers.py

```diff
@@ -689,6 +689,7 @@ class Meta:
             "image_src",
             "url",
             "offered_by",
+            "podcast",
```

Comment:
could we also add the `last_modified` field here, which is used for the published date? I suspect we'll want that in the UI

Reply:
Sounds good